README.md

```diff
@@ -6,7 +6,62 @@
 [![codecov](https://codecov.io/gh/material-motion/motion-interchange-objc/branch/develop/graph/badge.svg)](https://codecov.io/gh/material-motion/motion-interchange-objc)
 [![CocoaPods Compatible](https://img.shields.io/cocoapods/v/MotionInterchange.svg)](https://cocoapods.org/pods/MotionInterchange)
 [![Platform](https://img.shields.io/cocoapods/p/MotionInterchange.svg)](http://cocoadocs.org/docsets/MotionInterchange)
-[![Docs](https://img.shields.io/cocoapods/metrics/doc-percent/MotionInterchange.svg)](http://cocoadocs.org/docsets/MotionInterchange)
+
+"Magic numbers" — those lonely, abandoned values without a home — are often one of the first things
```

Comment:
Does this paragraph match the tone of Material Motion documentation? It feels much more playful/fun than the MDC READMEs.  I think that's absolutely OK if that's what we're going for, just pointing out the differences.

Reply:
I'm intentionally giving some more character to these libraries so that their content is a bit lighter to read and hopefully more memorable.

Reply:
Got it. Have fun with it! :D